server.js

```diff
@@ -14,15 +13,127 @@ mongoose.Promise = Promise
 const port = process.env.PORT || 8080
 const app = express()
 
+// specifying properties for the message, heart and createdAt
+// Date.now, creating date at the moment whenit is creating 
+
+const thoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: [true, "message is required"],
+    unique: true,
```

Comment:
Why can't I be happy for the same thing that others are? I think the message is not a good candidate for a unique property.